worker/src/ocall_bridge/component_factory.rs

```diff
@@ -38,30 +39,39 @@ use substratee_enclave_api::remote_attestation::RemoteAttestationCallBacks;
 /// Concrete implementation, should be moved out of the OCall Bridge, into the worker
 /// since the OCall bridge itself should not know any concrete types to ensure
 /// our dependency graph is worker -> ocall bridge
-pub struct OCallBridgeComponentFactory<N, B, W, E> {
```

Comment:
I don't think it makes sense that the OcallBridge gets so many parameters, also it does not make sense that the ocall bridge "decides" how sidechain blocks are handled. But with issue #https://github.com/integritee-network/worker/issues/301 this will need to be looked at anyway.

Reply:
I think I'd also prefer to keep these things more separate.